.cicd/supported_examples.json

```diff
@@ -172,5 +172,8 @@
   "utilities-examples/cloudify_terminal": [
     "linux-ssh.yaml",
     "fortigate.yaml"
+  ],
+    "scaling-example": [
+    "ansible.yaml"
```

Comment:
I don't really want to add this to testing yet, until we have all the other infras.

Reply:
It is just the blueprint validation.